tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -3,8 +3,12 @@
 import pytest
 from tempfile import NamedTemporaryFile
 
-from ocs_ci.framework.testlib import skipif_ocp_version, skipif_ocs_version, E2ETest
-from ocs_ci.framework.pytest_customization.marks import system_test
+from ocs_ci.framework.testlib import (
+    skipif_ocp_version,
+    skipif_ocs_version,
+    E2ETest,
```

Comment:
This is a functional test, so it should be manage test ideally.

Reply:
You can fix this in a subsequent PR.

Reply:
Done in https://github.com/red-hat-storage/ocs-ci/pull/7792